api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -14,8 +14,11 @@ option java_outer_classname = "WasmProto";
 option java_multiple_files = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
-// [#protodoc-title: Wasm]
-// [#extension: envoy.bootstrap.wasm]
```

Comment:
not sure if i misunderstand - but i think we dont want to lose the extension annotation

Reply:
oh i see its moved